roles/validate_http_store/tasks/main.yml

```diff
@@ -12,7 +12,7 @@
 
 - name: Retrieve file from http_store
   uri:
-    url: "{{ http_host }}/{{ test_file_name }}"
+    url: "{{ http_host }}:{{ http_port }}/{{ test_file_name }}"
```

Comment:
Move http_port into http_host so that it doesn't get added twice

Reply:
DONE